TEMPLATES.md

```diff
@@ -111,3 +111,10 @@ before gathering the artifacts.
 `ci-operator.openshift.io/always-show-output`:
 Will output the logs of all the containers in the pod, no matter what the exit code was.
 The value should be `true` to enable this feature.
+
+`ci-operator.openshift.io/always-show-container-output`:
+Will output the logs of the specified containers after the pod fails, if
+they are in a terminated state.
```

Comment:
Briefly describe the default state somewhere. What containers' output is captured by default?